packages/react-magma-dom/src/components/NavTabs/NavTab.tsx

```diff
@@ -89,6 +93,7 @@ const StyledTab = styled.a<{
   isActive?: boolean;
   isFullWidth?: boolean;
   isInverse?: boolean;
+  isFocused?: boolean;
```

Comment:
I don't think this needs to be passed to TabStyles if TabStyles doesn't use it, right?